src/api/controllers/getAuthorsController.js

```diff
@@ -0,0 +1,42 @@
+import models from '../models';
+import status from '../../helpers/constants/status.codes';
+
+const { Article } = models;
+const { User } = models;
+/**
+ *
```

Comment:
Description @Ramadhan0

Reply:
I changed the file name 
that's why this is not marked as resolved